### ISSUE TITLE
Share dataset

### ISSUE THREAD

**Issue by dreamflasher** [igorbrigadir/DownloadConceptualCaptions#6]
Would you be so kind to share the downloaded datasets? S3  would be ideal, alternatively torrent?

**Comment by igorbrigadir** [comment]
I'd love to but I only downloaded the 15k Validation set fully, for the 3 million training set i only ran HEAD requests to check the links. If you find or have a share of the actual downloaded data somewhere i'll put a link up!

**Comment by dreamflasher** [comment]
I crawled many of them, but it downloaded only 1% of the total, which seems a bit fishy to me, are all these links already offline? It's very sad that this dataset is not properly available, as the images obviously vanish over time.

**Comment by igorbrigadir** [comment]
That's very strange! Downloading only 1% is a bit too much decay - what did the final report look like in terms of errors and results? you can use https://github.com/igorbrigadir/DownloadConceptualCaptions/blob/master/downloaded_report.ipynb the same notebook to check

**Comment by dreamflasher** [comment]
I managed to download more afterall, regarding sharing the dataset: I guess the problem is that it's too big. Someone would need to host it, which they can't do because of copy right. Do you have any idea for that?

**Comment by igorbrigadir** [comment]
My immediate idea would be to host a torrent for the data?

Either way - if you have time, i'd love to get a copy of the `downloaded_report.ipynb` notebook when it's run on the full dataset - to get the stats of deleted / existing files, sizes etc.
